### PR TITLE
Remove unused parameter of GUIVolumeChange

### PR DESCRIPTION
**Pull request by Rui-Minetest**
"client" is unused.

### CODE REVIEW

File: src/guiVolumeChange.cpp [hunk 1]
```diff
@@ -52,23 +51,23 @@ GUIVolumeChange::~GUIVolumeChange()
 void GUIVolumeChange::removeChildren()
 {
 	{
-		gui::IGUIElement *e = getElementFromId(ID_soundText1);
-		if(e != NULL)
+		gui::IGUIElement* e = getElementFromId(ID_soundText1);
```
Comment (SmallJoker) [inline src/guiVolumeChange.cpp]: Please undo all these pointer changes, they were correct.

File: src/guiVolumeChange.cpp [hunk 2]
```diff
@@ -137,45 +136,50 @@ void GUIVolumeChange::drawMenu()
...
+		if (event.KeyInput.Key == KEY_RETURN && event.KeyInput.PressedDown)
 		{
 			quitMenu();
 			return true;
 		}
 	}
-	if(event.GUIEvent.EventType==gui::EGET_BUTTON_CLICKED)
+
+	if (event.GUIEvent.EventType == gui::EGET_BUTTON_CLICKED)
+	{
```
Comment (SmallJoker) [inline src/guiVolumeChange.cpp]: Please move the bracket to the upper line, so it looks like `if (condition) {`

File: src/guiVolumeChange.h [hunk 3]
```diff
@@ -30,8 +30,7 @@ class GUIVolumeChange : public GUIModalMenu
 public:
 	GUIVolumeChange(gui::IGUIEnvironment* env,
 			gui::IGUIElement* parent, s32 id,
-			IMenuManager *menumgr,
-			Client* client);
+			IMenuManager* menumgr);
```
Comment (SmallJoker) [inline src/guiVolumeChange.h]: Pointer on the wrong side

File: src/guiVolumeChange.cpp [hunk 4]
```diff
@@ -30,15 +30,14 @@ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
 
 #include "gettext.h"
 
-const int ID_soundText1 = 263;
-const int ID_soundText2 = 264;
+const int ID_soundText1      = 263;
+const int ID_soundText2      = 264;
 const int ID_soundExitButton = 265;
-const int ID_soundSlider = 266;
+const int ID_soundSlider     = 266;
```
Comment (nerzhul) [inline src/guiVolumeChange.cpp]: can you fixe thoses indents

**Comment by nerzhul** [comment]
Okay for me

File: src/guiVolumeChange.cpp [hunk 5]
```diff
@@ -53,22 +52,22 @@ void GUIVolumeChange::removeChildren()
 {
 	{
 		gui::IGUIElement *e = getElementFromId(ID_soundText1);
-		if(e != NULL)
+		if (e != NULL)
```
Comment (nerzhul) [inline src/guiVolumeChange.cpp]: in fact if you want to cleanup the code there, replace 

{
gui::IGUIElement *e = getElementFromId(ID_soundText1);
if (e != NULL)
e->remove()
}

with

if (gui::IGUIElement *e = getElementFromId(ID_soundText1))
    e->remove();

Comment (Rui-Minetest) [inline src/guiVolumeChange.cpp]: Changed. Thank you for your advice.

**Pull request merged**